test.sh

```diff
@@ -510,6 +517,119 @@ function test_example_programs() {
     "$BINDIR"/examples/splinterdb_custom_ipv4_addr_sortcmp_example
 }
 
+# ##################################################################
+# Smoke Tests: Run a small collection of fast-running unit-tests
+# ##################################################################
+function run_fast_unit_tests() {
```

Comment:
Most of this change is to do stuff like this ... refactoring inline calls to new functions.